products/pan-cortex-xdr.md

```diff
@@ -2,119 +2,158 @@
 title: Palo Alto Networks Cortex XDR agent
 category: app
 permalink: /cortex-xdr
+alternate_urls:
+-   /pan-xdr
+-   /cortexxdr
+-   /xdr
+-   /pan-cortex-xdr
 releasePolicyLink: https://www.paloaltonetworks.com/services/support/end-of-life-announcements/end-of-life-summary
 changelogTemplate: https://docs.paloaltonetworks.com/cortex/cortex-xdr/{{"__RELEASE_CYCLE__"
   | remove:'-' | replace:'.','-'}}/cortex-xdr-agent-release-notes/cortex-xdr-agent-release-information
+LTSLabel: CE
 activeSupportColumn: false
 releaseColumn: false
 releaseDateColumn: true
 eolColumn: Support Status
-alternate_urls:
--   /pan-xdr
--   /cortexxdr
--   /xdr
--   /pan-cortex-xdr
+
 releases:
+-   releaseCycle: "8.0"
+    eol: 2023-12-19
+    releaseDate: 2023-03-05
+    latestReleaseDate: 2023-03-05
+    latest: '8.0'
+    link: https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/8.0/Cortex-XDR-Agent-Release-Notes/Cortex-XDR-Agent-8.0-Release-Information
+
+-   releaseCycle: "7.9-ce"
+    lts: true 
+    eol: 2025-03-19
+    releaseDate: 2023-03-19
+    latestReleaseDate: 2023-03-19
+    latest: '7.9-ce'
+    link: https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/7.9ce/Cortex-XDR-Agent-Release-Notes/Cortex-XDR-Agent-7.9-CE-Release-Information
+
 -   releaseCycle: "7.9"
     eol: 2023-09-11
     releaseDate: 2022-12-04
     latestReleaseDate: 2023-03-19
     latest: '7.9-ce'
```

Comment:
With the addition of the 7.9-ce cycle, should `latest` be 7.9-ce for 7.9 cycle ?

Note that I do not think we will have the choice : even if we revert the version to `7.9` the automation will update it back to `7.9-ce`. So, if this is appropriate, maybe it would be best to remove the 7.9-ce cycle and mark the 7.9 as LTS (with `lts: 2023-03-19`) ?

Reply:
The [description](https://github.com/endoflife-date/endoflife.date/pull/2713#issuecomment-1476269742) says that they are two different cycles which have different support dates. So we have to list them separately.

If the automation threat them not as separate cycles we have to update the automation. @marcwrobel do you know the details for the automation for pan-cortex-xdr?

Reply:
The issue lies in the [latest.py](https://github.com/endoflife-date/endoflife.date/blob/master/_auto/latest.py) script. There is currently no mechanism to prevent it from updating a release when it detects a new version (and `7.9-ce` is a new version in the `7.9` cycle for this script).

The only thing I can think of to prevent this update is to "mark" this cycles and modify the script to skip marked cycles. But I am not fan of this mechanism. Will try to think of another solution.

Reply:
> The fix here would be to fix our automation so it actually gets the patch versions (which are hidden away across multiple pages) from the docs.

Did not thought about it, thanks @captn3m0.